OpenRA.Mods.Common/Scripting/Properties/CaptureProperties.cs

```diff
@@ -31,12 +30,21 @@ public CaptureProperties(ScriptContext context, Actor self)
 		[Desc("Captures the target actor.")]
 		public void Capture(Actor target)
 		{
-			var targetManager = target.TraitOrDefault<CaptureManager>();
-			if (targetManager == null || !targetManager.CanBeTargetedBy(target, Self, captureManager))
-				throw new LuaException($"Actor '{Self}' cannot capture actor '{target}'!");
+			if (!CanCapture(target))
+			{
+				Log.Write("lua", $"Actor '{Self}' cannot capture actor '{target}'!");
+				return;
+			}
```

Comment:
We don't/shouldn't throw on actions that can easily change runtime and are not controlled by Lua, see `Attack` for example. Throwing when there is an error independent from game state is ok Imo. (E.g. passing an invalid actor name to `CreateActor`.)

Reply:
Attack is literally the only other place where we log lua failure. So you haven't answered my question. 

> Is it the new convention to log failed commands?

Reply:
As I said, you don't want to log all failed API calls. Where there is a clear programming error we still want to throw. (E.g. wrong actor name.) Logging is better than silently failing Imo.

Reply:
But if its used often and expected to sometimes fail then it also doesn't make much sense to log.

Reply:
I think we can skip logging here because you can visually debug the script error when the actor does not capture?

Reply:
I don't think TC/RA/D2k missions are worth considering because they are *very* static and this will likely never be an issue unless it's a programming error like a wrong actor name.
What *could* be a valid use case are maps like the the Enhanced series for D2k, which add Lua goodies on top of a multiplayer ~~mess~~ experience. You don't want to be logging all failed attempts there, but also you can't easily debug anything visually and might need debug logs. 🤷‍♂️

Reply:
You could use the new `CanCapture` to debug log?

Reply:
Removed the extra logging.